### ISSUE TITLE
spyder find and replac

### ISSUE THREAD

**Issue by kirill-pinigin** [spyder-ide/spyder#6820]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/home/kirill/anaconda/lib/python2.7/site-packages/spyder/plugins/editor.py", line 2051, in replace
    editorstack.find_widget.show_replace()
  File "/home/kirill/anaconda/lib/python2.7/site-packages/spyder/widgets/findreplace.py", line 301, in show_replace
    self.show(hide_replace=False)
  File "/home/kirill/anaconda/lib/python2.7/site-packages/spyder/widgets/findreplace.py", line 283, in show
    self.refresh()
  File "/home/kirill/anaconda/lib/python2.7/site-packages/spyder/widgets/findreplace.py", line 320, in refresh
    self.find()
  File "/home/kirill/anaconda/lib/python2.7/site-packages/spyder/widgets/findreplace.py", line 441, in find
    total_matches=number_matches)
  File "/home/kirill/anaconda/lib/python2.7/site-packages/spyder/widgets/findreplace.py", line 575, in change_number_matches
    total_matches)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-1: ordinal not in range(128)

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 2.7.14
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.5.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.20.3 (OK)
numpy >=1.7       :  1.13.3 (OK)
sphinx >=0.6.6    :  1.6.3 (OK)
rope >=0.9.4      :  0.10.5 (OK)
jedi >=0.9.0      :  0.10.2 (OK)
psutil >=0.3      :  5.4.0 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.26.1 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0;<6.0:  5.4.1 (OK)
pylint >=0.25     :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.8). Please update by running in a terminal (xterm, konsole, etc)

    conda update spyder

**Comment by CAM-Gerlach** [comment]
For reference, a duplicate of #6187 and fixed in Spyder 3.2.7.
